packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -86,6 +109,11 @@ export default defineConfig({
       config.env.ADDRESS = testWalletAddress
       config.env.PRIVATE_KEY = testWallet.privateKey
       config.env.INFURA_KEY = process.env.NEXT_PUBLIC_INFURA_KEY
```

Comment:
The tests failed to run for me until I added `NEXT_PUBLIC_INFURA_KEY` to `.e2e.env`, too. We should document this

Reply:
It should be read from `.env` as well but I've added it to `.sample.e2e.env` now too